torch/_dynamo/variables/builder.py

```diff
@@ -2384,6 +2384,9 @@ def handle_traced_output(example_value, tx, proxy, options, subclass_type, targe
     ):
         set_example_value(proxy.node, example_value)
         return ConstantVariable.create(example_value, **options)
+    elif isinstance(example_value, str) and (proxy.node.target in ["hex"]):
```

Comment:
Fwiw the condition here is too grabby: it will grab hex methods on non symfloat. But the terminal state is everything should go this path so whatever